scripts/apps/archive/index.js

```diff
@@ -155,6 +155,26 @@ angular.module('superdesk.apps.archive', [
                     return authoring.itemActions(item).duplicate;
                 }]
             })
+            .activity('duplicateTo', {
+                label: gettext('Duplicate To'),
+                icon: 'copy',
+                monitor: true,
+                controller: ['data', 'send', function(data, send) {
+                    send.allAs([data.item], 'duplicateTo');
+                }],
+                filters: [
+                    {action: 'list', type: 'archive'},
+                    {action: 'list', type: 'archived'}
+                ],
+                keyboardShortcut: 'ctrl+alt+t',
```

Comment:
LGTM, only one minor thing that, `ctrl+alt+t` interfering and opening terminal window when FF/Chrome running under ubuntu(just an alt. suggestion if shortcut is required: ctrl+alt+c / ctrl+alt+r), also alt+k shortcut list need to be update for `Monitoring / List actions` section